tests/test__mongomock.py

```diff
@@ -2542,6 +2542,15 @@ def test__aggregate_count(self):
             {'$count': 'my_count'}
         ])
 
+    def test__aggregate_facet(self):
+        self.cmp.do.insert_many([
+            {'_id': i} for i in range(5)
+        ])
+        self.cmp.compare.aggregate([
+            {'$facet': {'pipeline_a': [{'$count': 'my_count'}],
```

Comment:
Please fix indentation so that pipeline_a and pipeline_b are both alone on their own line.